update-format-crau/src/delta_update.rs

```diff
@@ -176,9 +184,17 @@ pub fn verify_sig_pubkey(digest: &[u8], sig: &Signature, pubkeyfile: &str) -> Op
     // in case of production update payloads. However, we do not explicitly check
     // for a signature version, as the number could differ in some cases.
     debug!("supported signature version: {:?}", sig.version());
+    if sig.version() == 1 {
+        info!("unsupported signature version {:?}, nothing to verify", sig.version());
+        return Ok(sig.data.clone().unwrap().into_boxed_slice());
+    }
+
     let sigvec = match &sig.data {
-        Some(sigdata) => Some(sigdata),
-        _ => None,
+        Some(sigdata) => sigdata,
+        _ => {
+            info!("empty signature data, nothing to verify");
+            return Ok(sig.data.clone().unwrap().into_boxed_slice());
```

Comment:
We never create unsigned payload and I don't see a need of supporting this because it's rather a risk of accepting wrongly because of a logic error.

Reply:
Done